app/src/main/java/com/kickstarter/viewmodels/AccountViewModel.kt

```diff
@@ -94,10 +105,22 @@ interface AccountViewModel {
             return this.progressBarIsVisible
         }
 
+        override fun showEmailErrorIcon(): Observable<Boolean> = this.showEmailErrorIcon
+
         override fun success(): BehaviorSubject<String> {
             return this.success
         }
 
+        private fun showEmailErrorImage(userPrivacy: UserPrivacyQuery.Data?): Boolean? {
+            return if (userPrivacy?.me()?.isCreator!! && !userPrivacy.me()?.isDeliverable!! || !userPrivacy.me()?.isEmailVerified!!) {
+                return true
+            } else if (!userPrivacy.me()?.isDeliverable!! && !userPrivacy.me()?.isEmailVerified!!) {
```

Comment:
For this one, I thought for backers, we are only uninterested in if the email is undeliverable and we don't care if it's unverified.

We should also have a test for this case. That we don't show the icon for unverified backers.

Reply:
fixed